src/common/GitHubVS.ruleset

```diff
@@ -37,6 +37,7 @@
         <Rule Id="CA1501" Action="None" />
         <Rule Id="CA1502" Action="None" />
         <Rule Id="CA1506" Action="None" />
+        <Rule Id="CA1701" Action="None" />
```

Comment:
At this point, could we remove `<Rule Id="CA1303" Action="None" />` so that we get build failures if we add literal strings?